salt/metalk8s/volumes/unprepared/init.sls

```diff
@@ -8,5 +8,5 @@ Clean up backing storage for {{ volume }}:
     - name: {{ volume }}
 {%- else %}
 Volume "{{ volume }}" not found in pillar:
-  test.fail_without_changes: []
+  test.succeed_without_changes: []
```

Comment:
Is there some way to still render some kind of 'warning' message in the state's output, e.g. using `test.configurable_test_state` and setting the `comment` to something descriptive?

While indeed this should be 'success', it's also something that should not occur all too frequently, and something we may want to report to a user (beyond the `name` of the state).

Reply:
For me you should log warning in jinja so that you will have real warning output in log and in output https://docs.saltstack.com/en/latest/ref/modules/all/salt.modules.logmod.html
Because put something directly in the comment of the state result is not really usefull if it's success nobody will read it IMO.

Reply:
Then do both. Keep in mind that, when the operator invokes Salt, no human will see any Salt logs (except when explicitly looking at the logs on the minion system at that point in time).

Reply:
Done